.pylintrc

```diff
@@ -10,7 +10,7 @@ extension-pkg-whitelist=numpy,tensorflow,scipy,networkx,strawberryfields
 # (useful for modules/projects where namespaces are manipulated during runtime
 # and thus existing member attributes cannot be deduced by static analysis. It
 # supports qualified module names, as well as Unix pattern matching.
-ignored-modules=numpy,tensorflow,scipy,networkx,strawberryfields,strawberryfields.parameters
+ignored-modules=numpy,tensorflow,scipy,networkx,strawberryfields,strawberryfields.parameters,collections.abc
```

Comment:
For some reason, pylint/codefactor was saying 'No Sequence in module collections.abc'